Makefile

```diff
@@ -65,46 +65,65 @@ create : delete build build-burstserver
 
 build :
 	# build the WebAssembly
-	@rm -f wasm_header_poc.wasm
+	@rm -f burst_header.wasm
 	@cargo build --release --target=wasm32-unknown-unknown
-	@cp target/wasm32-unknown-unknown/release/wasm_header_poc.wasm .
+	@cp target/wasm32-unknown-unknown/release/burst_header.wasm .
 
 delete:
 	# delete the cluster (if exists)
 	@# this will fail harmlessly if the cluster does not exist
 	-kind delete cluster
 
-deploy :
-	# TODO deploy the app
+deploy : build
 
 	@kubectl apply -f deploy/ngsa-memory/ngsa-memory.yaml
 	@kubectl apply -f deploy/ngsa-memory/ngsa-gw.yaml
 
 	@kubectl wait pod --for condition=ready --all --timeout=60s
 
-	@kubectl apply -f deploy/loderunner/loderunner.yaml
+	# create HPA for ngsa deployment for testing
+	@kubectl autoscale deployment ngsa --cpu-percent=50 --min=1 --max=2
+	@kubectl wait pod --for condition=ready --all --timeout=60s
+
+	# Patching Istio ...
+	@./patch.sh
+
+	@# add config map
+	@kubectl create cm burst-wasm-filter --from-file=burst_header.wasm
+
+	@# patch any deployments
+	@# this will create a new deployment and terminate the old one
+	@kubectl patch deployment ngsa -p '{"spec":{"template":{"metadata":{"annotations":{"sidecar.istio.io/userVolume":"[{\"name\":\"wasmfilters-dir\",\"configMap\": {\"name\": \"burst-wasm-filter\"}}]","sidecar.istio.io/userVolumeMount":"[{\"mountPath\":\"/var/local/lib/wasm-filters\",\"name\":\"wasmfilters-dir\"}]"}}}}}'
+
+	@# turn the wasm filter on for each deployment
+	@kubectl apply -f deploy/ngsa-memory/filter.yaml
+
+	@kubectl wait pod --for condition=ready --all --timeout=60s
 
 check :
 	# get the metrics
 	@curl -q http://${K8s}/burstmetrics/default/ngsa
 	@echo ""
 
 	# check the healthz endpoint
-	@http http://${K8s}/memory/healthz
+	# @http http://${K8s}/memory/healthz
+	@http http://${K8s}/memory/healthz User-Agent:HTTPie/1.0.3
 
```

Comment:
@bartr, I modified this to be explicit on the header, because that's the only way it worked for me.  Maybe a different version of httpie?  In any case, this fixes the issue I had.  I think we should probably just change this to the target User-Agent value and make that the default in the filter.yaml.

Reply:
I think we should change the code to use starts_with and not specify the version. Otherwise we're going to have to redeploy ...